doc/explanation/configure-storage.rst

```diff
@@ -0,0 +1,177 @@
+
+.. image:: figures/configure-storage-lvm.png
+   :alt:
+
+The LVM is a system of managing logical volumes, or filesystems, that is much
+more advanced and flexible than the traditional method of partitioning a disk
+into one or more segments and formatting that partition with a filesystem. It
+can be used to combine several disks into one larger pool of storage but it
+offers advantages even in a single disk system, such as snapshots and easy
+resizing of logical volumes.
+
+As with RAID, a LVM volume group can be created out of entire disks or
+unformatted partitions. Select the "Create LVM volume group" button to open
+the creation dialog.
+
+Once a volume group has been created, it can be divided into named logical
+volumes which can then be formatted and mounted. It generally makes sense to
+leave some space in the volume group for storage of snapshots and creation of
+more logical volumes as needed.
+
+The server installer does not supported configuring any of the many, many
+options that LVM supports when creating volume groups and logical volumes.
+
+Selecting boot devices
+======================
+
+.. image:: figures/configure-storage-boot-devices.png
+   :alt:
+
+On all architectures other than s390x, the bootloader needs to be installed to
+a disk in such a way that the system firmware can find it on boot. By default,
+the first device to have a partition created on it is selected as a boot device
+but this can be changed later.
+
+On amd64 and arm64 systems, multiple disks can be selected as boot devices,
+which means a system can be configured so that it will continue to boot after
+a failure of any one drive (assuming the root filesystem is placed on a RAID).
+The bootloader will be installed to each of these drives, and the operating
+system configured to install new versions of GRUB to each drive as it is
+updated.
+
+amd64 systems use GRUB as the bootloader. amd64 systems can boot in either UEFI
+or legacy (sometimes called "BIOS") mode (many systems can be configured to
+boot in either mode) and the bootloader is located completely differently in
+the two modes.
+
+Legacy mode
+-----------
+
+In legacy mode, the bootloader is read from the first "sector" of a hard drive
+(exactly which hard drive is up to the system firmware, which can usually be
+configured in a vendor-specific way). The installer will write GRUB to the
+start of all disks selected as a boot devices. As GRUB does not entirely fit
+in one sector, a small unformatted partition is needed at the start of the
+disk, which will automatically be created when a disk is selected as a boot
+device (a disk with an existing GPT partition table can only be used as a boot
+device if it has this partition).
+
+UEFI mode
+---------
+
+In UEFI mode, the bootloader loaded from a "EFI System Partition" (ESP), which
+is a partition with a particular type GUID. The installer automatically creates
+a 512MiB ESP on a disk when it is selected as a boot device and will install
```

Comment:
I'd like to omit the size default, it's subject to frequent debate and also scales to available space.
If we do leave a number in, the current minimum is 538 MiB.

Reply:
Could you suggest a wording you'd rather use instead, and I'll change it to that? (I think it'd be better to leave the minimum number in there if possible, it's always helpful to give people even an order-of-magnitude estimate of how much space will be required)

Reply:
maybe just the word "approximately" thrown in there?

> The installer automatically creates an ESP on a disk, approximately 512MiB in size, when it is selected as a boot device ...

Reply:
I've made a slight tweak. I've gone with the "minimum 538" option as I think it's clearer for users.